### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by artcz**

### CODE REVIEW

File: src/config.toml.example [hunk 1]
```diff
@@ -0,0 +1 @@
+pretalx-token = "this is pretalx token - for submissions"
```
Comment (egeakman) [inline src/config.toml.example:1]: Maybe we can remove this if we're not planning to use a config file?

**Review by egeakman — APPROVED** [review]
LGTM! Maybe we can remove the unused imports as well?

**Comment by egeakman** [comment]
See #2

**Comment by artcz** [comment]
Merging this for now, so we can base PRs on main

**Pull request merged**